reference/pipeline/stages/index.md

```diff
@@ -176,7 +176,53 @@ Wait a specified period of time before proceeding. You can choose to manually
 skip some or all of the wait period during execution.
 
 ### Webhook
-Run a Webhook job.
+Make an API call to an external system as part of your pipeline. Supply the URL
```

Comment:
I would break the paragaph after the first sentence. That way, you get just the definition, then the details.